development/webpack/test/webpack.config.test.ts

```diff
@@ -231,7 +233,7 @@ ${Object.entries(env)
     assert.deepStrictEqual(manifestPlugin.options.description, null);
     assert.deepStrictEqual(manifestPlugin.options.zip, true);
     assert(manifestPlugin.options.zipOptions, 'Zip options should be present');
-    assert.strictEqual(manifestPlugin.options.transform, undefined);
+    assert.notEqual(manifestPlugin.options.transform, undefined);
```

Comment:
same as https://github.com/MetaMask/metamask-extension/pull/29696/files#r1917494955

Reply:
reverted in b2b58623abf962e3e283a60f82178490a5f4dae3